src/ui/SwiftOutputChannel.ts

```diff
@@ -65,6 +66,7 @@ export class SwiftOutputChannel {
 
     private sendLog(line: string) {
         this.channel.append(line);
+        this.logStore.append(line);
```

Comment:
suggestion: what I've seen before is the file log have all diagnostics, including the verbose ones. i.e. the "diagnostics" setting would only affect output channel verbosity

Reply:
There isn't really a verbose/non-verbose concept in the extension at the moment; everything just goes to the output channel